src/auro-alaska.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable one-var */
+// Copyright (c) 2020 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+import { html, css } from "lit-element";
+import { classMap } from 'lit-html/directives/class-map';
+import BaseIcon from "./baseIcon";
+
+import as400 from '@alaskaairux/icons/dist/restricted/AS-400_es6.js';
+import as300 from '@alaskaairux/icons/dist/restricted/AS-300_es6.js';
+import as200 from '@alaskaairux/icons/dist/restricted/AS-200_es6.js';
+import as100 from '@alaskaairux/icons/dist/restricted/AS-100_es6.js';
+import asTag400 from '@alaskaairux/icons/dist/restricted/AS-tagline-400_es6.js';
+import asTag300 from '@alaskaairux/icons/dist/restricted/AS-tagline-300_es6.js';
+import asTag200 from '@alaskaairux/icons/dist/restricted/AS-tagline-200_es6.js';
+import asTag100 from '@alaskaairux/icons/dist/restricted/AS-tagline-100_es6.js';
+
+// Import touch detection lib
+// import "focus-visible/dist/focus-visible.min.js";
+import styleCss from "./alaskaStyle-css.js";
+
+// See https://git.io/JJ6SJ for "How to document your components using JSDoc"
+/**
+ * auro-alaska provides users a way to use the Auro Icons by simply passing in the category and name.
+ *
+ * @attr {Boolean} alaska - Set value for default alaska airlines logo
+ * @attr {Boolean} alaskaOfficial - Set value for alaska airlines logo with official tagline
+ * @slot - Hidden from visibility, used for a11y if icon description is needed
+ */
+
+// build the component class
+class AuroAlaska extends BaseIcon {
+  constructor() {
+    super();
+
+    this.privateDefaults();
+  }
+
+  /**
+   * @private internal defaults
+   * @returns {void}
+   */
+  privateDefaults() {
+    this.uri = 'https://unpkg.com/@alaskaairux/icons@latest/dist';
+    this.zero = 0;
+    this.no = 1;
+    this.sm = 107;
+    this.md = 191;
+    this.lg = 527;
+  }
+
+  // function to define props used within the scope of this component
+  static get properties() {
+    return {
+      ...super.properties,
+      alaska: {
```

Comment:
Do we need this prop any more with auro-alaska split out into its own thing?

Reply:
I guess I'm not sure what the difference is between

```html
<auro-alaska></auro-alaska>
<auro-alaska alaska></auro-alaska>
<auro-alaska alaskaOfficial></auro-alaska>
```

Seems like the API could be named more clearly.

Reply:
The difference is between the logo type. 

Alaska is only the Alaska name. AlaskaOfficial is the Alaska name with the Airlines tagline. This is how the files supplied by the design team named them.

Reply:
> Do we need this prop any more with auro-alaska split out into its own thing?

The value is used in the `firstUpdated()` lifecycle method. The use is that there are two different logos that point to two different SVG files. 

```
  async firstUpdated() {
    const iconContainer = await this.shadowRoot.querySelectorAll('.icon');
    const iconWidth = iconContainer[this.zero].clientWidth;

    if (this.alaskaOfficial) {
      this.alaska = false;
    } else {
      this.alaska = true;
    }

    if (this.alaska) {
      this.alaskaLogoDef(iconWidth);
    } else if (this.alaskaOfficial) {
      this.alaskaLogoTagDef(iconWidth);
    }
  }
```

I see what you mean about the API not being clear. It wasn't until the end that I decided to make the `alaska` logo the default. I also need to clean up the API docs. Yeah, lazy is not an excuse.